cli.js

```diff
@@ -6,29 +6,51 @@ const chalk = require('chalk');
 const npmName = require('npm-name');
 
 const cli = meow(`
-	Usage
-	  $ npm-name <name>
+  Usage
+    $ npm-name <name>
 
-	Examples
-	  $ npm-name chalk
-	  ${logSymbols.error} ${chalk.bold('chalk')} is unavailable
-	  $ npm-name unicorn-cake
-	  ${logSymbols.success} ${chalk.bold('unicorn-cake')} is available
+  Examples
+    $ npm-name chalk
+    ${logSymbols.error} ${chalk.bold('chalk')} is unavailable
+    $ npm-name unicorn-cake
+    ${logSymbols.success} ${chalk.bold('unicorn-cake')} is available
+    $ npm-name chalk,unicorn-cake
+    ${logSymbols.error} ${chalk.bold('chalk')} is unavailable
+    ${logSymbols.success} ${chalk.bold('unicorn-cake')} is available
 
-	Exits with code 0 when the name is available or 2 when taken
+  Exits with code 0 when the name is available or 2 when taken
 `, {
-	string: ['_']
+  string: ['_']
 });
 
-const input = cli.input[0];
+let input = cli.input[0];
+
+if (input.indexOf(',') > -1) {
+  input = input.split(',');
+}
 
 if (!input) {
-	console.error('Package name required');
-	process.exit(1);
+  console.error('Package name required');
+  process.exit(1);
 }
 
 npmName(input).then(available => {
-	const name = chalk.bold(input);
-	console.log(available ? `${logSymbols.success} ${name} is available` : `${logSymbols.error} ${name} is unavailable`);
-	process.exit(available ? 0 : 2);
+  let exitWithError = available;
+
+  if (available instanceof Array) {
+    for (let i = 0, len = available.length; i < len; i++) {
```

Comment:
Well since the `input` array and the `available` array have matching indexes, how else would we get that index? We could use `for..in`, but then the body has to be wrapped in an if statement (http://eslint.org/docs/rules/guard-for-in).